src/Microsoft.AspNetCore.Mvc.Core/RedirectToRouteResult.cs

```diff
@@ -27,10 +27,28 @@ public RedirectToRouteResult(object routeValues)
             string routeName,
             object routeValues,
             bool permanent)
+            : this(routeName, routeValues, permanent, fragment: null)
+        {
+        }
+
+        public RedirectToRouteResult(
```

Comment:
My doc comment and if-industrious comments in `RedirectToActionResult` apply here too.